ac/ac-autocode/package.json

```diff
@@ -12,10 +12,10 @@
     "react-ace": "^5.9.0",
     "react-bootstrap": "^0.32.1",
     "react-dom": "^16.2.0",
-    "styled-components": "^3.1.6",
-    "victory": "^0.25.3"
+    "styled-components": "^3.2.0",
+    "victory": "^0.25.6"
   },
   "devDependencies": {
-    "nps": "5.7.1"
```

Comment:
I'm having trouble again today while creating a new operator because when we do this kind of updates, we do not keep the templates up-to-date. Please update the dependencies in the templates too